administrator/components/com_cookiemanager/src/Model/ConsentsModel.php

```diff
@@ -121,7 +121,7 @@ protected function getListQuery()
 		if (!empty($search))
 		{
 			$search = $db->quote('%' . str_replace(' ', '%', $db->escape(trim($search), true) . '%'));
-			$query->where('(a.ccuuid LIKE ' . $search . ')');
+			$query->where('(a.ccuuid LIKE ' . $search . 'or a.uuid LIKE ' . $search . ')');

```

Comment:
@vorayash can you use the `andWhere` function here instead of writing the `or` manually?

Reply:
Like this **$query->where('(a.ccuuid LIKE ' . $search . ')') -> andWhere('(a.uuid LIKE ' . $search . ')');** ?

Reply:
@vorayash That is correct.

Reply:
Actually I will need to use orWhere

Reply:
$query->where('(a.ccuuid LIKE ' . $search . ')') -> orWhere('(a.uuid LIKE ' . $search . ')');

Reply:
@vorayash Why do you think you need to use the `orWhere`? This will add an `AND` constraint and the search needs an `OR` constraint.

Reply:
https://user-images.githubusercontent.com/72155000/168483607-ab9646e8-0606-43ec-8001-116b9328324c.mp4

Reply:
Look at your video, you are adding `()` yourself to the query, those should not be there. What are the queries you get?

Reply:
This also not working '$query->where('a.ccuuid LIKE' . $search) -> andWhere('a.uuid LIKE ' . $search)'

Reply:
> This also not working '$query->where('a.ccuuid LIKE' . $search) -> andWhere('a.uuid LIKE ' . $search)'
This will treat as 'select ..... where ccuuid LIKE search and uuid LIKE search'

Reply:
@vorayash I had a closer look and realized we have nothing in `where` yet at this point. So the way to this is `$query->where(['a.ccuuid LIKE ' . $search, 'a.uuid LIKE ' . $search], 'OR');` This will create the correct query. Can you update your PR?

Reply:
Sure. Thanks @roland-d.